assets/css/tailwind.css

```
@@ -28,16 +28,31 @@
   --color-warning: rgb(var(--theme-color-warning));
   --color-info: rgb(var(--theme-color-info));
   --color-blue: #245A7D;
+
+  /* Brand typeface. Noto Sans has no CJK coverage, so Noto Sans JP is stacked explicitly. */
+  --font-sans: "Noto Sans", "Noto Sans JP", -apple-system, BlinkMacSystemFont,
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)